cmd/lncli/cmd_open_channel.go

```diff
@@ -424,7 +424,8 @@ func openChannelPsbt(ctx *cli.Context, client lnrpc.LightningClient,
 		return fmt.Errorf("opening stream to server failed: %v", err)
 	}
 
-	if err := signal.Intercept(); err != nil {
+	signal, err := signal.Intercept()
```

Comment:
To avoid colliding with the package name, perhaps name it `shutdownInterceptor`?

Reply:
Fixed.